micro-infra-spring-config/src/test/groovy/com/ofg/infrastructure/property/ConfigLocationsTest.groovy

```diff
@@ -0,0 +1,46 @@
+package com.ofg.infrastructure.property
+
+import spock.lang.Specification
+
+class ConfigLocationsTest extends Specification {
+
+    static final File COMMON_DIR = new File('/props/common/')
```

Comment:
I haven't checked it, but it should work also on Windows.

File contructor calls normalize method from FileSystem and Win32FileSystem implementation can deal with slashes.

Nevertheless it is not well documented, so I can create files without slashes.